Core/src/org/sleuthkit/autopsy/filequery/FileSearch.java

```diff
@@ -304,9 +311,111 @@ private static TextSummary getDefaultSummary(AbstractFile file) {
         } catch (TskCoreException ex) {
             logger.log(Level.WARNING, "Error getting children for file: " + file.getId(), ex);
         }
-        image = image == null ? image  : image.getScaledInstance(ImageUtils.ICON_SIZE_MEDIUM, ImageUtils.ICON_SIZE_MEDIUM,
+        image = image == null ? image : image.getScaledInstance(ImageUtils.ICON_SIZE_MEDIUM, ImageUtils.ICON_SIZE_MEDIUM,
                 Image.SCALE_SMOOTH);
-        return new TextSummary(getFirstLines(file), image, countOfImages);
+        String summaryText = null;
+        if (file.getMd5Hash() != null) {
+            try {
+                summaryText = getSavedSummary(Paths.get(Case.getCurrentCaseThrows().getCacheDirectory(), "summaries", file.getMd5Hash() + "-default-" + PREVIEW_SIZE + "-translated.txt").toString());
+            } catch (NoCurrentCaseException ex) {
+                logger.log(Level.WARNING, "Unable to retrieve saved summary case not open", ex);
```

Comment:
Perhaps add a dash between summary and case?

Reply:
I changed it to "Unable to save translated summary. No case is open." and the other logger message similarly.